tests/models/test_dagbag.py

```diff
@@ -765,9 +765,13 @@ def subdag_1():
 
             return dag
 
-        test_dag = nested_subdag_cycle()
-        # coherence check to make sure DAG.subdag is still functioning properly
-        assert len(test_dag.subdags) == 6
+        with pytest.warns(
+            RemovedInAirflow3Warning,
+            match="This class is deprecated. Please use `airflow.utils.task_group.TaskGroup`.",
+        ):
+            test_dag = nested_subdag_cycle()
+            # coherence check to make sure DAG.subdag is still functioning properly
+            assert len(test_dag.subdags) == 6
```

Comment:
Does this line need to be inside the context block?

Reply:
Thanks. Made the change.